src/main/CommandLine.cpp

```diff
@@ -1867,43 +1867,32 @@ ParserWithValidation
 applyLoadModeParser(std::string& modeArg, ApplyLoadMode& mode)
 {
     auto validateMode = [&] {
-        if (iequals(modeArg, "soroban"))
+        if (iequals(modeArg, "ledger-limits"))
         {
-            mode = ApplyLoadMode::SOROBAN;
+            mode = ApplyLoadMode::LIMIT_BASED;
             return "";
         }
-        if (iequals(modeArg, "classic"))
```

Comment:
curious, why no more classic?

Reply:
Just to simplify the code and usage modes. Classic payments can still be generated via the config flag, but there is really no need for introducing different modes. Classic-only mode would just set soroban txs to 0, soroban-only mode would set classic txs to 0, and mixed mode will have both non-0. E.g. in the 'history' example we generate both classic and soroban txs.